tests/helper/utils.py

```diff
@@ -5,6 +5,21 @@
 import subprocess
 
 
+def read_file_remote(client, file, remove_comments=False)-> list:
+    """ Read a file from the remote file system and returns it for 
+        proper processing trunecated.
+    """
+    status, output = execute_remote_command(client, f"cat {file}", skip_error=True)
+    assert status == 0, f"Error reading {file}" 
+    to_return = output.split("\n")
+    # Check if we might have an emtory entry.
+    if "" in to_return:
+        to_return.remove("")
+    if remove_comments: 
+        return [line for line in to_return if '#' not in line]
+    return to_return
```

Comment:
```suggestion
   return [line for line in output.split("\n") if line and if not line.startswith("#")]
```
Your version would also remove lines like:

`woah this is code # and this is an inline comment!`
 
Also I would love to see what this function does for the file "/dev/urandom" or "/dev/zero" :D

Reply:
Good catch. I'm going to check your solution.